src/windows/win-crowdstrike-fix-bootloop-v2.ps1

```diff
@@ -4,6 +4,51 @@
 $partitionlist = Get-Disk-Partitions
 $actionTaken = $false
 
+function CleanUpRegtransmsAndTxrblfFiles 
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$GuidSuffix,
+        [Parameter(Mandatory = $true)]
+        [string]$DriveLetter
+    )
+
+    Log-Info "Deleting regtrans-ms and txr.blf files under config\TxR for Windows Server 2016 or newer version..."
+    Log-Info "Checking Windows Build number..."
+    $regKey = "HKLM:\temp_software_hive_$GuidSuffix\Microsoft\Windows NT\CurrentVersion"
+    $currentBuild = (Get-ItemProperty $regKey -Name CurrentBuild).CurrentBuild
```

Comment:
check for exit code/error?

Reply:
Will it throw if this property does not exist.

Reply:
If the querying failed, it will not throw error but return $currentBuild as $null. I just added the logic to check if the $currentBuild is $null in the next line.